models/get-models.js

```diff
@@ -16,3 +16,28 @@ exports.selectArticle = (article_id) => {
       return rows[0];
     });
 };
+
+exports.selectArticles = () => {
+  return db
+    .query("SELECT * FROM comments")
+    .then(({ rows }) => {
+      const commentData = {};
+
+      for (let i = 0; i < rows.length; i++) {
+        const currentArticle_id = rows[i].article_id;
+        commentData[currentArticle_id] =
+          commentData[currentArticle_id] + 1 || 1;
+      }
+      return commentData;
+    })
+    .then((commentData) => {
+      return db.query("SELECT * FROM articles").then(({ rows }) => {
+        rows.forEach((article) => {
+          article.comment_count = commentData[article.article_id] || 0;
+          delete article.body;
+        });
+        rows.sort((a, b) => b.created_at - a.created_at);
```

Comment:
Same for this - could psql just do this for us so we don't have to do this manually? Think about what key words you've used in your complex queries so far